Jenkinsfile

```diff
@@ -53,6 +53,24 @@ pipeline {
             make install
           '''
         }
+        dir ('proxygen') {
+          checkout([$class: 'GitSCM',
+          branches: [[name: '*/master']],
+          extensions: [[$class: 'SubmoduleOption',
+                        disableSubmodules: false,
+                        parentCredentials: false,
+                        recursiveSubmodules: true,
+                        reference: '',
+                        trackingSubmodules: false]], 
+          userRemoteConfigs: [[url: 'git@github.com:facebook/proxygen.git']]])
+          sh '''
+            cd proxygen
+            ./build.sh -m
```

Comment:
Can you pass an option here saying to install to a local directory directory? (like how `libff` is using `install` subdirectory for installing components)